unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -429,18 +428,18 @@ protected void HideAllAgentRenderers()
             return reachablePos;
         }
 
-        public void GetReachablePositions(int maxStepCount = 0) {
-            if(maxStepCount != 0) {
-                reachablePositions = getReachablePositions(1.0f, maxStepCount);
+        public void GetReachablePositions(int? maxStepCount = null) {
+            Vector3[] reachablePositions;
+            if (maxStepCount != null) {
+                reachablePositions = getReachablePositions(maxStepCount: (int) maxStepCount);
```

Comment:
Here I'd also prefer `maxStepCount.Value`.

Reply:
Fixed in https://github.com/allenai/ai2thor/pull/628/commits/f4f78b083cfbda37c743642548cacc5c72606492.